src/SFA.DAS.AdminService.Web/Views/MergeOrganisations/SetSecondaryEpaoEffectiveToDate.cshtml

```diff
@@ -0,0 +1,73 @@
+﻿@model SFA.DAS.AdminService.Web.ViewModels.Merge.SetSecondaryEpaoEffectiveToDateViewModel
+
+<a class="govuk-back-link" asp-controller="MergeOrganisations" asp-action="MergeOverview"> Back </a>
+
+<main class="govuk-main-wrapper " id="main-content" role="main">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+
+            <partial name="_ValidationSummaryPartial" />
+
+            <h1 class="govuk-heading-xl">
+                Choose an effective to date for the secondary EPAO
+            </h1>
+
+            <div class="govuk-inset-text">
+                The secondary EPAO will be removed from the Register of EPAOs on the Effective to date. <br />
+                They will no longer be able to use their Assessor account.
```

Comment:
Content query: preventing access is a future requirement, not currently in the ACs. Is this content OK?